packages/data-point/package.json

```diff
@@ -80,20 +79,5 @@
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
     }
-  },
-  "jest": {
-    "coveragePathIgnorePatterns": [
-      "/node_modules/",
-      "/test/utils/",
-      "/test/definitions/"
-    ]
-  },
-  "lint-staged": {
-    "*.js": [
-      "prettier-standard",
-      "standard",
-      "git add",
-      "jest --bail --findRelatedTests"
-    ]
```

Comment:
Why was this removed?

Reply:
it is redundant because this config is in the main package.json

Reply:
Ah, OK!